packages/arb-ts/src/lib/l2Bridge.ts

```diff
@@ -66,15 +60,15 @@ export class L2Bridge {
     )
   }
 
-  public async getERC20LlAddress(erc20L2Address: string) {
-    return await this.arbERC20Bridge.customToken(erc20L2Address)
+  public getERC20L1Address(erc20L2Address: string) {
+    return this.arbERC20Bridge.customToken(erc20L2Address)
```

Comment:
This seems off. Presumably it should use the calculated address if it's not custom